Makefile

```diff
@@ -156,6 +156,12 @@ build-test-e2e: build-services
 build-python-wheels:  ## Build all Python wheels
 	./pants filter --target-type=python_distribution :: | xargs ./pants package
 
+.PHONY: build-services-local
+build-services-local: 
+	$(WITH_LOCAL_GRAPL_ENV) \
+	$(MAKE) build-services
+	$(MAKE) push-local
```

Comment:
Since `build-services` already uses `WITH_LOCAL_GRAPL_ENV`, we don't gain anything by doing it here, too.

At that point, it's probably better to write this target as:

```make
build-services-local: build-services
    $(MAKE) push-local
```
But then we have the question: is `build-services-local` really just an alias for `push-local`? Should we just get rid of `build-services-local` and have `push-local` depend on `build-services`? Is there a scenario where you would legitimately need to call `push-local` without building the services?

Reply:
> Since build-services already uses WITH_LOCAL_GRAPL_ENV, we don't gain anything by doing it here, too.

I don't think that's true. If I run `make build-services` the docker build does not see the `DOCKER_REGISTRY` environment variable.